### PR TITLE
Adds portainer index file and associated images

### PR DESCRIPTION
**Pull request by goatatwork**

### CODE REVIEW

**Comment by ajeetraina** [comment]
Thanks @goatatwork for the submission. Added few missing pieces like:
- Updating sidebar.js
- Modifying to index-portainer.mdx
- Initial syntax to mdx file for Portainer

**Comment by ajeetraina** [comment]
@simonprickett Can you please review the content structure.

File: docs/create/portainer/index-portainer.mdx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+---
+
+
+
+
+Redis is an in-memory data structure store, used as a distributed, in-memory key–value database, cache and message broker, with optional durability. Redis supports different kinds of abstract data structures, such as strings, lists, maps, sets, sorted sets, HyperLogLogs, bitmaps, streams, and spatial indices. [~Wikipedia](https://en.wikipedia.org/wiki/Redis)
+
+You can’t travel far in the modern software world without finding that you need an in-memory data store. Today, the answer to “Which data store?” is often Redis. Also today, the answer to “How to Redis?” can be Portainer.
+
+In this blog we'll take you through using Portainer to turn up Redis in three common scenarios.
```
Comment (simonprickett) [inline docs/create/portainer/index-portainer.mdx]:
```suggestion
In this blog we'll take you through using Portainer to set up Redis in three common scenarios.
```

File: docs/create/portainer/index-portainer.mdx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+![create_namespace](images/redis1.png)
+
+Now that we have a namespace for Redis to operate in, let’s install it.
+
+In the navigation menu on the left, click **“Helm”**. If you haven’t already done so, add the Bitnami charts repository by typing https://charts.bitnami.com/bitnami into the **“Additional repositories”** form, and then click **“Add repository”**.
+
+![helm](images/redis2.png)
+
+As long as the Bitnami charts repository has been added, you should see a list of Charts on this page. Find **"Redis"** and click on it.
```
Comment (simonprickett) [inline docs/create/portainer/index-portainer.mdx]:
```suggestion
Once the Bitnami charts repository has been added, you should see a list of Charts on this page. Find **"Redis"** and click on it.
```

File: docs/create/portainer/index-portainer.mdx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+Note, you will see redis-cluster listed as an option. The redis-cluster Helm chart configures a six node cluster; three masters, and three slaves. The redis Helm chart we will use configures a much simpler three node cluster; one master, and two slaves. There are a number of other differences between these two Helm charts. For a complete list of differences, Bitnami has a good description [here](https://docs.bitnami.com/kubernetes/infrastructure/redis/get-started/compare-solutions/)
```
Comment (simonprickett) [inline docs/create/portainer/index-portainer.mdx]:
```suggestion
Note, you will see redis-cluster listed as an option. The redis-cluster Helm chart configures a six node cluster; three masters, and three slaves. The redis Helm chart we will use configures a much simpler three node cluster; one master, and two slaves. There are a number of other differences between these two Helm charts. For a complete list of differences, Bitnami has a good description [here](https://docs.bitnami.com/kubernetes/infrastructure/redis/get-started/compare-solutions/).
```

File: docs/create/portainer/index-portainer.mdx [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+When it’s finished, you can head to the Applications page by clicking Applications in the navigation menu on the left. When Kubernetes is finished bringing up Redis, you will see the Status as **“Ready”**.
+
+![redis-ready](images/redis5.png)
+
+Note that what is deployed is a STATEFULSET application, which means it persists data. You can see the volumes which have been created (which use the default storage class of your system) by clicking **"Volumes"**.
+
+![volumes](images/redis6.png)
+
+See that each pod has its own copy/replica of the DB content, and note that it defaults to 8GB size. If you need to change this, then its line 409 in the values file of the HELM deployment.
```
Comment (simonprickett) [inline docs/create/portainer/index-portainer.mdx]:
```suggestion
See that each pod has its own copy/replica of the DB content, and note that it defaults to 8GB in size. If you need to change this, then its line 409 in the values file of the HELM deployment.
```

File: docs/create/portainer/index-portainer.mdx [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+Long live Docker Swarm! There is still plenty of Swarm in the wild and Portainer is managing a lot of it. Let’s bring up Redis in Swarm.
+
+For the purposes of this demo, we will use a Portainer App Template to deploy Redis, and this assumes you are using Portainer’s provided App Template Repo. You can check that under “Settings”.
+
+![settings](images/redis10.png)
+
+Now that you know you are good to go,
+
+- Click "App Templates" in the navigation menu on the left.
+- Find and click on Redis Cluster
```
Comment (simonprickett) [inline docs/create/portainer/index-portainer.mdx]:
```suggestion
- Find and click on Redis Cluster.
```

File: docs/create/portainer/index-portainer.mdx [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+- Provide a SECURE password for **Redis**.
+- Click the **“Deploy the stack”** button.
+
+![deploy-stack](images/redis12.png)
+
+You will be taken to the “Stacks list” page and will see the new stack named “redis”. Click on “redis” to see information about this stack. Like this:
+
+![stack-info](images/redis13.png)
+
+To test, Expand one of the services that you see on the stack details page (above). Then click on the container’s “exec console” icon.
```
Comment (simonprickett) [inline docs/create/portainer/index-portainer.mdx]:
```suggestion
To test, expand one of the services that you see on the stack details page (above). Then click on the container’s “exec console” icon.
```

File: docs/create/portainer/index-portainer.mdx [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+- Give the application a name, in this case "redis".
+- Click on “Show advanced options”.
+- Set the port to map. In this example, the Docker **host**'s port 6379 is forwarded to the **container**'s port 6379, Redis’s default port for most communications.
+- Click on the “Deploy the container” button.
+
+![deploy-container](images/redis18.png)
+
+That’s it. You can test in the same way as before.
+
+![contaienr-container](images/redis19.png)
```
Comment (simonprickett) [inline docs/create/portainer/index-portainer.mdx]:
```suggestion
![container-container](images/redis19.png)
```

**Review by simonprickett — APPROVED** [review]
@ajeetraina looks good left a couple of small suggestions.

**Pull request merged**